### PR TITLE
Added AutoChargeStationSequence to drive to charging station and call…

### PR DESCRIPTION
**Pull request by avakimc**
… balancing command

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -40,4 +46,8 @@ public RobotContainer() {
   private void configureBindings() {
     driverController.a().whileTrue(new InstantCommand(intake::deploy, intake));
   }
+
+  public static Command getAutonomousCommand(){
+    return new AutoChargeStationSequence(true, new DriveSubsystem());
```
Comment (balghane) [inline src/main/java/frc/robot/RobotContainer.java]: Can you make `true` a named variable in this function just so people don't have to go into AutoChargeStationSequence to see what the true is referring to?

Comment (balghane) [inline src/main/java/frc/robot/RobotContainer.java]: Can you instantiate a driveSubsystem as a member variable in this class, and then just pass that instance into this function?

File: src/main/java/frc/robot/commands/AutoChargeStationSequence.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Calibrations;
+import frc.robot.subsystems.drive.DriveSubsystem;
+
+public class AutoChargeStationSequence extends SequentialCommandGroup {
```
Comment (balghane) [inline src/main/java/frc/robot/commands/AutoChargeStationSequence.java:15]: Can you add a top-level `/** */` style comment for what this auto does

**Review by balghane — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (avakimc) [reply src/main/java/frc/robot/RobotContainer.java]: Okay, when I did this I also changed the method so it's not longer static.

**Review by balghane — APPROVED** [review]

File: src/main/java/frc/robot/Calibrations.java [hunk 3]
```diff
@@ -72,4 +72,14 @@ public static final class SwerveModuleCalibrations {
   /** Intake positions in degrees */
   public static final double INTAKE_STARTING_POSITION_DEGREES = PLACEHOLDER_DOUBLE,
       INTAKE_DEPLOYED_POSITION_DEGREES = PLACEHOLDER_DOUBLE;
+
+  /** Time to drive the robot to approach the charging station in auto in seconds */
+  public static final double DRIVE_TIME_AUTO_SECONDS = PLACEHOLDER_DOUBLE;
```
Comment (balghane) [inline src/main/java/frc/robot/Calibrations.java]: We can remove this one

Comment (avakimc) [reply src/main/java/frc/robot/Calibrations.java]: OK

**Pull request merged**